app/assets/javascripts/beak/widgets/ractives/status-popup.coffee

```diff
@@ -0,0 +1,56 @@
+RactiveStatusPopup = Ractive.extend({
+
+  data: -> {
+    isUp:              false
+    hasWorkInProgress: false
+    statusTimer:       0
+  }
+
+  on: {
+    'init': () ->
+      if @get('hasWorkInProgress')
+        @showStatus()
+
+    'close-status': () ->
+      @closeStatus()
+  }
+
+  showStatus: () ->
+    @set('isUp', true)
+    timer = 5
```

Comment:
I really think the pop-up disappears too quickly.  Even doubling its duration might not make it appear for long enough, but it really seems to go away faster than a person who was reading it for the first time would be able to read it.

Reply:
:+1: I'll bump it up to 20 seconds.